src/Microsoft.Health.Fhir.SqlServer/Features/Watchdogs/DefragWatchdog.cs

```diff
@@ -212,11 +212,11 @@ private async Task<(long groupId, long jobId, long version, int activeDefragItem
             (long groupId, long jobId, long version) id = (-1, -1, -1);
             try
             {
-                JobInfo job = (await queueClient.EnqueueAsync(QueueType, new[] { "Defrag" }, null, true, false, cancellationToken)).FirstOrDefault();
+                var jobs = await queueClient.EnqueueAsync(QueueType, new[] { "Defrag" }, null, true, false, cancellationToken);
```

Comment:
The previous call is `FirstOrDefault`, can nulls still be returned here?

Reply:
Previous null was only in case of empty list. No, job info cannot be null.